client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1618,6 +1618,7 @@
   "messages.confirm-not-fully-supplied_one": "There is {{count}} line in this requisition that has not been fully supplied to the customer. \n\nWould you still like to continue?",
   "messages.confirm-not-fully-supplied_other": "There are {{count}} lines in this requisition that have not been fully supplied to the customer. \n\nWould you still like to continue?",
   "messages.confirm-patient-retrieval": "Are you sure you want to retrieve patient data for {{name}}?",
+  "messages.confirm-purchase-order-status-as": "Confirm status as $t({{status}})?",
```

Comment:
nit: duplicate translation value - but I guess could be different in other languages?